### PR TITLE
Rename MNID module functions

### PR DESCRIPTION
**Pull request by ender503**
To improve the code readability, rename the MNID module function with
meaningful names, and add code documentation.

The function name are renamed as following:
 - checksum() -> generateChecksum()
 - encode() -> encodeToMnid()
 - decode() -> decodeFromMnid()
 - encodeToDid() -> encodeToDid()
 - decodeFromDid() -> decodeFrom()

### CODE REVIEW

File: packages/did/src/IdenityRegistry.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 const tic = require('tic.api.js');
 const mamClient = require('mam.tools.js');
 
-const { encodeToDid, decodeFromDid } = require('./mnid');
+const { encodeTangleId, decodeTangleId } = require('./mnid');
```
Comment (jserv) [inline packages/did/src/IdenityRegistry.js]: Avoid using `TangleId`. Instead, always use `TangleID`. Be aware of the sequence.

Comment (jserv) [inline packages/did/src/IdenityRegistry.js]: Why would you change?

Comment (jserv) [reply packages/did/src/IdenityRegistry.js]: Tangle is one of the blockchain implementations, fitting the requirement of DID.

Comment (ender503) [reply packages/did/src/IdenityRegistry.js]: How about rename them to the `encodeDid` and `decodeDid`? 🤔

Comment (jserv) [reply packages/did/src/IdenityRegistry.js]: Either introduce new layer for encapsulating Tangle or rename back to encodeToDid/decodefromDid. I prefer the later.

Comment (ender503) [reply packages/did/src/IdenityRegistry.js]: Okay, I'll change them as following example:
     - encode() -> encodeToMnid()
     - decode() -> decodeFromMnid()
     - encodeToDid() -> encodeToDid()
     - decodeFromDid() -> decodeFrom()

Comment (ender503) [reply packages/did/src/IdenityRegistry.js]: The function names are updated. 👍

**Pull request merged**